<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sysInfo</class>
 <widget class="QWidget" name="sysInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>处理器信息</string>
        </property>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CPU名称:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CPU类型:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CPU频率:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>120</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Cache大小:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_CPUname">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>421</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_CPUtype">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>431</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_CPUfrequency">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>431</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_Cachecapacity">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>431</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>操作系统信息</string>
        </property>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>操作系统类型:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>操作系统版本:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_26">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>90</y>
           <width>67</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>GCC版本:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_OStype">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>30</y>
           <width>411</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_OSversion">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>60</y>
           <width>411</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_GCCversion">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>90</y>
           <width>411</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>未知</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
